libs/SalesforceSDK/src/com/salesforce/androidsdk/config/RuntimeConfig.java

```diff
@@ -52,6 +53,7 @@
 	private RuntimeConfig(Context ctx) {
 		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
 			configurations = getRestrictions(ctx);
+            isManaged = hasRestrictionsProvider(ctx);
```

Comment:
This is the way to test that one is managed - the restrictions bundle is non-null on lollipop whether or not one is managed